code/clue.js

```diff
@@ -1,59 +1,309 @@
+  color: '#d1a00d',
+  description: 'A dignified, dapper and dangerous military man',
+  age: 55,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel'
+}
+
+// *******  WEAPONS  *********
 
 const rope = {
-  name: 'rope',
-  weight: 10
+  name: 'Rope',
+  weight: 1.5,
+  image: 'assets/Rope_Full.jpg'
+}
+
+const knife = {
+  name: 'Knife',
+  weight: 0.2,
+  image: 'assets/Dagger_Full.jpg'
+}
+
+const candlestick = {
+  name: 'Candlestick',
+  weight: 12,
+  image: 'assets/candlestick.jpg'
+}
+
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 30,
+  image: 'assets/dumbbell.jpg'
+}
+
+const poison = {
+  name: 'Poison',
+  weight: 0.02,
+  image: 'assets/poison.jpg'
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 3,
+  image: 'assets/axe.jpg'
+}
+
+const bat = {
+  name: 'Bat',
+  weight: 2,
+  image: 'assets/bat.jpg'
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 8,
+  image: 'assets/trophy.jpeg'
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 0.7,
+  image: 'assets/pistol.jpg'
+}
+
+// *****  THE ROOMS  ****** //
+
+const diningRoom = {
+  name: 'Dining Room',
+  image: 'assets/dining-room.jpg'
+}
+
+const conservatory = {
+  name: 'Conservatory',
+  image: 'assets/conservatory.jpg'
+}
+
+const kitchen = {
+  name: 'Kitchen',
+  image: 'assets/kitchen.jpg'
+}
+
+const study = {
+  name: 'Study',
+  image: 'assets/study.jpg'
+}
+
+const library = {
+  name: 'Library',
+  image: 'assets/library.jpg'
+}
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  image: 'assets/billiard-room.jpg'
+}
+
+const lounge = {
+  name: 'Lounge',
+  image: 'assets/lounge.jpg'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const ballroom = {
+  name: 'Ballroom',
+  image: 'assets/ballroom.jpg'
+}
+
+const hall = {
+  name: 'Hall',
+  image: 'assets/hall.jpg'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const spa = {
+  name: 'Spa',
+  image: 'assets/spa.jpg'
+}
+
+const livingRoom = {
+  name: 'Living Room',
+  image: 'assets/living-room.jpg'
+}
+
+const observatory = {
+  name: 'Observatory',
+  image: 'assets/observatory.jpeg'
+}
+
+const theater = {
+  name: 'Theater',
+  image: 'assets/theater.jpg'
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  image: 'assets/guest_house.jpg'
+}
+
+const patio = {
+  name: 'Patio',
+  image: 'assets/patio.jpg'
+}
+
+// ***** ARRAYS *****
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  mrsPeacock,
+  professorPlum,
+  missScarlet,
+  colMustard,
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio
 ]
 
-const weapons = []
+// *******  ADDING COLOR TO THE OBJECTS  *********
+
+weapons.forEach(weapon =>
+  weapon.color = '#394042')
```

Comment:
Interesting, so you decided to add the data to the object and make them all the same color. 
It feels like this could have been done in CSS unless you had plans to change the color for some objects